src/middlewared/middlewared/plugins/system.py

```diff
@@ -121,6 +121,10 @@ class Config:
             Bool('anonstats'),
             Str('sed_user', enum=['USER', 'MASTER']),
             Str('sed_passwd', private=True),
+            Str('sysloglevel', enum=['F_EMERG', 'F_ALERT', 'F_CRIT', 'F_ERR',
+                                     'F_WARNING', 'F_NOTICE', 'F_INFO',
+                                     'F_DEBUG', 'F_IS_DEBUG']),
+            Str('syslogserver'),
```

Comment:
You will also need to change system.general to reroute the syslog settings to use system.advanced.

Also please add a deprecation warning over there when these settings are used in system.general.